e2e/replay/msg.go

```diff
@@ -0,0 +1,50 @@
+package replay
+
+import (
+	"encoding/json"
+
+	"github.com/0xPolygon/pbft-consensus"
+)
+
+// Message is a struct that represents a single json object in .flow file
+type Message struct {
+	To      pbft.NodeID      `json:"to"`
+	Message *pbft.MessageReq `json:"message"`
+}
+
+// NewMessageReq creates a new message to be written to .flow file
+func NewMessageReq(to pbft.NodeID, message *pbft.MessageReq) *Message {
```

Comment:
Rename this to `NewMessage`. (also consider making it private/package visibility)